packages/valory/skills/decision_making_abci/tests/test_write_stream_preparation.py

```diff
@@ -58,7 +58,6 @@ class BaseWriteStreamPreparationTest:
     def create_write_stream_object(self, write_stream_preparation_class):
         """Create the write stream object."""
         synchronized_data = MagicMock()
-        synchronized_data.centaurs_data = DUMMY_CENTAURS_DATA
```

Comment:
@annasambrook this is the problematic part. I have removed the tests that require this code temporarily to double-check that this resolves the issue on CI.

Reply:
It does :smiley: